relayer/src/api_server.rs

```diff
@@ -52,11 +51,11 @@ pub(crate) fn init_web_server(
     txn_queue: Arc<RwLock<TxnQueue>>,
 ) -> Result<task::JoinHandle<Result<(), std::io::Error>>, tide::Error> {
     let mut web_server = tide::with_state(WebState { txn_queue });
-    web_server.with(TraceMiddleware::new());
     web_server.at("/healthcheck").get(healthcheck);
     web_server.at("/submit").post(submit_endpoint);
     let port = std::env::var("PORT").unwrap_or_else(|_| DEFAULT_RELAYER_PORT.to_string());
     let addr = format!("0.0.0.0:{}", port);
+    tracing::info!("RELAYER ADDRESS {}", addr);
```

Comment:
This is not the `init_web_server` called by the minimal-relayer service.

Reply:
Thanks @nyospe , I will close the PR then.